site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -37,6 +37,8 @@ public function run() {
             case 'import_teams':
                 $this->importTeams();
                 break;
+            case 'randomize_team_rotating_sections':
+                $this->randomizeTeamRotatingSections();
```

Comment:
You should add a `break;` here to speed up the execution time

Reply:
Yes, this change has been made.  Thanks!